graphistry/feature_utils.py

```diff
@@ -137,29 +195,34 @@ def assert_imported():
 #
 #      _featurize_or_get_edges_dataframe_if_X_is_None
 
-FeatureEngineConcrete = Literal["none", "pandas", "dirty_cat", "torch"]
+FeatureEngineConcrete = Literal["none", "pandas", "dirty_cat", "torch", "cu_cat"]
 FeatureEngine = Literal[FeatureEngineConcrete, "auto"]
 
 
 def resolve_feature_engine(
     feature_engine: FeatureEngine,
 ) -> FeatureEngineConcrete:  # noqa
 
-    if feature_engine in ["none", "pandas", "dirty_cat", "torch"]:
+    if feature_engine in ["none", "pandas", DIRTY_CAT, "torch", CUDA_CAT]:
         return feature_engine  # type: ignore
+    elif feature_engine in ["cuda", "gpu"]:
```

Comment:
why both `"cuda"` and `"gpu"`, is one an alias for another? can we have just one?

Reply:
also, `FeatureEngine` type defintion on :199 does not allow `"cuda", "gpu"`, so we'd need to add there

for consistency, i'd expect this to be also added to umap: https://github.com/graphistry/pygraphistry/blob/db31c0ae0e3af655c6dd33155fca793e6c3ef167/graphistry/umap_utils.py#L93